### ISSUE TITLE
Toilet admission counts towards ride income

### ISSUE THREAD

**Issue by Gymnasiast** [OpenRCT2/OpenRCT2#21912]
This makes ride income scenarios easier.

I think it would make more sense to have it count against shop sales.

**Comment by ZehMatt** [comment]
I agree, well one might argue that the visit to the toilet can be a wild ride for some.

**Comment by 733737** [comment]
duplicate of https://github.com/OpenRCT2/OpenRCT2/issues/17372

**Comment by MASHzj** [comment]
Counting it under shop sales would make shop income scenarios easier, so I doubt if that would make it better.

**Comment by cheweytoo** [comment]
It's a water ride, obviously.

I don't play scenarios much, so I can't really gauge how much "shop income" scenarios would be affected by this.
I definitely agree that shop sales make more sense though.

**Comment by goodingr** [comment]
I'll try this
